src/main/java/tds/exam/results/mappers/OpportunityMapper.java

```diff
@@ -141,10 +139,7 @@ private static void mapExamItemsAndResponsesToOpportunity(final ExpandableExam e
                     scoreInfo.setScorePoint(scoreString);
                     scoreInfo.setMaxScore(String.valueOf(assessmentItem.getMaxScore()));
                     scoreInfo.setScoreStatus(scoreStatus);
-
-                    if (score.getScoringDimensions().isPresent()) {
-                        scoreInfo.setScoreDimension(score.getScoringDimensions().get());
-                    }
+                    scoreInfo.setScoreDimension(score.getScoringDimensions());
```

Comment:
With the latest exam client, this is no longer an Optional<String> property, just a String property.